packages/pwa-kit-react-sdk/setup-jest.js

```diff
@@ -6,16 +6,9 @@
  */
 /* eslint-env jest */
 
-import Enzyme from 'enzyme'
 import 'regenerator-runtime/runtime'
-
-// DANGEROUS: this enzyme React 17 adapter is unofficial
-// because the official adaptor is still in development
-// see https://github.com/enzymejs/enzyme/issues/2429
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-
-Enzyme.configure({adapter: new Adapter()})
-
+import '@testing-library/jest-dom'
+require('@testing-library/jest-dom/extend-expect')
```

Comment:
```suggestion
import '@testing-library/jest-dom/extend-expect'
```

Reply:
Looks like we don't need this import, import jest-dom above is enough. I've removed this line